pkg/log/log.go

```diff
@@ -76,7 +77,7 @@ func (l Logger) WithFields(fields ...zap.Field) Logger {
 // ErrorFilterContextCanceled wraps Logger.Error() and will filter error log when error is context.Canceled
 func (l Logger) ErrorFilterContextCanceled(msg string, fields ...zap.Field) {
 	for _, field := range fields {
-		if field.Key == "error" && field.String == context.Canceled.Error() {
+		if field.Key == "error" && strings.Contains(field.String, context.Canceled.Error()) {
```

Comment:
Can this handle `zap.Error` (`filed.String` is empty) rather than `log.ShortError`?

Reply:
No, this can't handle. I have updated the dealing for such situations.